amida-tech/indaba-client
# Feature/inba 961 + 986 remove grommet syling

#### What does this PR do?

Removes the Grommet package, and all remaining references to grommet in the SCSS and JS.

#### Related JIRA tickets:

INBA-961

#### How should this be manually tested?
Fundamentally, it requires a comparative side-by-side test against staging because so many files were changed. Check the following list comprehensively:

PM USER:
1) The All Users menu
2) The User Menu Modal (Accessible by clicking any user) Check all Tabs
3) Flag management screen
4) The Users tab of project management
5) The Users tab of the project creation wizard
6) Moving a user in and out of a staging slot in project management and creation
7) Subjects screen and list
8) All editable text inputs (test for activity, and when you enter an incorrect value or no value)
9) Creating and editing a survey
10) Task details modal (assign a user to a stage and click the ellipses)
11) Date pickers

USER:
1) The user profile (icon at top left)     
2) All editable text inputs
3) User flag screen
4) Survey screen (take a survey)

#### More Instruction

I fully expect I missed some stuff, so please point anything out you find that's off from Staging, and let me know so I can correct.
